src/openscad.cc

```diff
@@ -1041,6 +1054,7 @@ int main(int argc, char **argv)
 		}
 		else {
 			PRINTB("Unknown --export-format option '%s' ignored. Use -h to list available options.", tmp_format.c_str());
+			return 1;
```

Comment:
Per recent discussion.  Now, in command or GUI mode if --export-format is invalid, exit program.